content/docs/server_setup/server_software.mdx

```diff
@@ -0,0 +1,222 @@
+
+<Callout type="info" title="Note">
+  Due to NeoForge being a relatively new server software, there is not as many mods that completely support it compared to the likes of Forge and Fabric.
+  However, many mods are slowly being updated to support NeoForge, and many popular modpacks exist for it.
+</Callout>
+
+<Card href="https://neoforged.net/" title="Download">
+  Download a copy of the latest NeoForge server software via [neoforged.net](https://neoforged.net/).
+</Card>
+
+## Fabric
+Similar to Forge and NeoForge, Fabric is a modded server software (and API). Like Forge & NeoForge, it allows you to add new content (such as mobs, biomes, generation etc) to the game with ease.\
+It is known for being more lightweight with a more extensive array of performance mods compared to Forge, such as Lithium, Moonrise, Starlight, etc.
+
+<Card href="https://fabricmc.net/use/installer/" title="Download">
+  Download a copy of the latest Fabric server software (installer) via [fabricmc.net](https://fabricmc.net/use/installer/).
+</Card>
+<br></br>
+
+# Proxy Servers
+Proxies are server softwares that allow you to connect multiple servers together, making it more seamless for switching between servers / gamemodes.\
+These are known as **"networks"**.\
+Proxies are also incredibly lightweight, with small CPU and RAM usage (typically less than 1-2GB of RAM) however this can vary depending on your player count and proxy plugins.
+
+## Velocity (recommended)
+[Velocity](https://papermc.io/software/velocity) is a modern, high-performance, and extremely flexible Minecraft proxy, maintained by the [PaperMC Team](https://papermc.io/).
+It is relatively new, but has quickly become the most popular proxy server software due to its performance and flexibility.
+
+<Callout type="info" title="Note">
+    Velocity makes use of its own API, meaning all Bungeecord plugins do **not** function on Velocity, although there are workaround plugins that introduce some degree of compatibility between the two plugin platforms.\
+    One example of this is [Snap](https://hangar.papermc.io/Phoenix616/Snap).<br></br>
+
+    Ultimately Velocity is a far more modern and superior proxy software compared to Waterfall and Bungeecord, unless you make use of plugins that only have Bungeecord support.
+</Callout>
+
+<Card href="https://jar.setup.md/download/velocity/latest/latest" title="Download">
+  Download a copy of the latest Velocity proxy software via [jar.setup.md](https://jar.setup.md/).
+</Card>
+
+## Velocity-CTD
+[Velocity-CTD](https://github.com/GemstoneGG/Velocity-CTD) is a fork of [Velocity](#velocity-recommended) that adds additional features and optimizations to the software.\
+It is fully open source and adds Quality of Life commands and more support for backend server softwares - particularly modded servers which are notoriously difficult when within a network.
```

Comment:
softwares

Reply:
Resolved in [d20c33d](https://github.com/admincraft-community/admincraftwiki/pull/1/commits/d20c33d060a328b2cce5bbeb24c70aee63e38311)